packages/zmarkdown/config/remark.js

```diff
@@ -317,10 +317,6 @@ const remarkConfig = {
     inlineMathDouble: true,
   },
 
-  katex: {
-    inlineMathDoubleDisplay: true,
```

Comment:
pourquoi retirer celui-là?

Reply:
It doesn't exist anymore, the behaviour targeted by this option is now the default; rehype-katex decided to use remark-math's `inlineMathDouble` option (indirectly) to detect the option instead.